chat_shell/chat_shell/services/chat_service.py

```
@@ -182,20 +226,58 @@ async def _process_chat(
+                                document_ids=request.document_ids,
+                            )
+                            if kb_content:
+                                # Check if the content contains relevant information for Q3
+                                if (
+                                    "Q3" not in kb_content
+                                    and "第三季度" not in kb_content
+                                ):
+                                    logger.warning(
+                                        "[CHAT_SERVICE] Knowledge base does not contain Q3 2025 content. Available content may not be relevant to Q3 queries."
+                                    )
+                                    system_prompt = f"{system_prompt}\n\n# Knowledge Base Context\n\n{kb_content}\n\n注意：知识库中未找到2025年Q3（第三季度）的工作周报内容，当前提供的是其他季度的信息。"
```

Ruff (0.14.10)

String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)